public/app/core/services/keybindingSrv.ts

```diff
@@ -166,156 +170,158 @@ export class KeybindingSrv {
   }
 
   setupDashboardBindings(dashboard: DashboardModel) {
-    this.bind('mod+o', () => {
```

Comment:
Consider making the isInAnIframe change in this.bind() instead of everywhere this.bind() is used?

Reply:
Check moved to the bind function